contentcuration/contentcuration/serializers.py

```diff
@@ -148,7 +149,8 @@ class FileSerializer(BulkSerializerMixin, serializers.ModelSerializer):
     language = LanguageSerializer(many=False, required=False, allow_null=True)
     display_name = serializers.SerializerMethodField('retrieve_display_name')
     id = serializers.CharField(required=False)
-    preset = FormatPresetSerializer(many=False)
+    preset = FormatPresetSerializer(many=False, read_only=True)
+    # preset = serializers.PrimaryKeyRelatedField(many=False, queryset=FormatPreset.objects.all())
```

Comment:
Initially I'd just replaced `FormatPresetSerializer` with a `PrimaryKeyRelatedField`, but then I moved to change the superclass of `FormatPresetSerializer`.